app/models/user.rb

```diff
@@ -42,6 +42,10 @@ def name
     end
   end
 
+  def is_admin?
```

Comment:
Shouldn't this be called `admin?`?

See https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/PredicateName

```suggestion
  def admin?
```